src/modules/search/components/ClientSearch.tsx

```diff
@@ -91,11 +75,6 @@ export const CLIENT_COLUMNS: {
     key: 'age',
     render: (client) => asClient(client).age,
   },
-  linkedName: {
-    header: 'Name',
-    key: 'name',
-    render: (client) => <ClientName client={asClient(client)} linkToProfile />,
-  },
   linkedNameNewTab: {
```

Comment:
I removed those that weren't used anymore, but there are still a few here (like `linkedNameNewTab`) that are used in out-of-scope table that I didn't update.